src/System.Net.Http.WinHttpHandler/tests/UnitTests/WinHttpResponseHeaderReaderTest.cs

```diff
@@ -0,0 +1,182 @@
+// Copyright (c) Justin Van Patten. All rights reserved.
```

Comment:
See the discussion here: https://github.com/dotnet/coreclr/pull/1329#discussion_r36218185

This has been done elsewhere in the coreclr and corefx repos.